CODIGO/TileEngine.bas

```diff
@@ -1311,10 +1305,11 @@ Sub RenderScreen(ByVal tilex As Integer, ByVal tiley As Integer, ByVal PixelOffs
     'Effects Update
     Call Effect_UpdateAll
     
-    If ClientSetup.ProyectileEngine = True Then
-        Dim J As Integer
-        
+    If ClientSetup.ProyectileEngine Then
+                            
         If LastProjectile > 0 Then
+            Dim J As Long ' Long siempre en los bucles es mucho mas rapido                       
```

Comment:
Can you explain why a Long is faster in a loop? I don't know why and I can't find any reason for that.